platform/telemetry/src/TelemetrySubscription.js

```diff
@@ -239,6 +239,10 @@ define(
 
             initialize();
             this.unlistenToMutation = addMutationListener();
+
+            // Expose makeDatum for TelemetryHandler to use, but not
+            // as part of public API
+            this.makeDatum = makeDatum;
```

Comment:
Agreed, and think we should adopt the `@private` approach generally (since adding JSDoc to private methods will facilitate static time-checking as well, and provides explicit disambiguation of public and private API).